app/api/sync/specs/uploadRoute.spec.ts

```diff
@@ -26,6 +26,13 @@ jest.mock(
   }
 );
 
+jest.mock(
+  '../../auth/headersMiddleware.ts',
+  () => (_req: Request, _res: Response, next: NextFunction) => {
+    next();
+  }
+);
+
```

Comment:
again, is this needed ? can we avoid it somehow ?

Reply:
Removed